src/EnergyPlus/DataLoopNode.hh

```diff
@@ -262,12 +309,26 @@ namespace DataLoopNode {
 			TempSetPointHi( TempSetPointHi ),
 			TempSetPointLo( TempSetPointLo ),
 			Height( Height ),
+			SchedOutAirDryBulb ( SchedOutAirDryBulb ),
```

Comment:
If they aren't being used, they should be removed.  I thought someone went through and removed all the unused ones.....